scripts/prompt_injection/api.py

```
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
+"""
+Prompt Injection Detection API
+
+This module provides a simple FastAPI server for detecting prompt injection attacks.
+"""
+
+import os
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)